### PR TITLE
Match nav panel icons 

### PR DESCRIPTION
**Pull request by almedina-ms**
## Summary of the Pull Request

Match the options and icons in the nav panel

image

### CODE REVIEW

File: src/TerminalSettings/TerminalSettings/MainPage.xaml [hunk 1]
```diff
@@ -19,24 +19,47 @@
...
+                </NavigationViewItem.Icon>
+                <TextBlock Tag="Keybindings_Page">Command Palette</TextBlock>
+            </NavigationViewItem>
+
+            <NavigationViewItem Icon="Shop"
                                     Tag="Keybindings_Nav">
-                <TextBlock Tag="Keybindings_Page">Keybindings</TextBlock>
+                <TextBlock Tag="Keybindings_Page">Marketplace</TextBlock>
             </NavigationViewItem>
+
```
Comment (vineeththomasalex) [inline src/TerminalSettings/TerminalSettings/MainPage.xaml:62]: Nit: These extra empty lines are unnecessary and likely isn't the format in the other Terminal xaml files. Would be good to check the other files.

**Review by vineeththomasalex — APPROVED** [review]
Only comments are about formatting, which we can update later. Can merge this in to get the tab format in.

File: src/TerminalSettings/TerminalSettings/MainPage.xaml [hunk 2]
```diff
@@ -19,24 +19,47 @@
...
+                <NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE790;"></FontIcon>
+                </NavigationViewItem.Icon>
+                <TextBlock Tag="ColorSchemes_Page">Appearance</TextBlock>
+            </NavigationViewItem>
+
             <NavigationViewItem Tag="Profiles_Nav">
                 <NavigationViewItem.Icon>
-                    <FontIcon Glyph="&#xE756;" />
+                    <FontIcon Glyph="&#xE770;" />
```
Comment (carlos-zamora) [inline src/TerminalSettings/TerminalSettings/MainPage.xaml:34]: It'd be _nice_ if you put a comment on each of the icons for what they actually are.
```suggestion
                    <!--- Color Palette Icon --->
                    <FontIcon Glyph="&#xE770;" />
```
and similarly throughout

**Review by carlos-zamora — APPROVED** [review]
Nit

File: src/TerminalSettings/TerminalSettings/MainPage.xaml (hunk 2 above)
Comment (vineeththomasalex) [reply src/TerminalSettings/TerminalSettings/MainPage.xaml:34]: Let's push this in for now and format code before pushing in to the main branch.

**Pull request merged**